src/eme.js

```diff
@@ -168,6 +163,11 @@ export const makeNewRequest = (player, requestOptions) => {
       keySession.addEventListener(KEY_STATUSES_CHANGE, (event) => {
         let expired = false;
 
+        // Protect from race conditions causing the player to be disposed.
+        if (player.isDisposed()) {
+          return;
+        }
```

Comment:
Using `isDisposed()` is a somewhat cleaner way to check if we can trigger events.